### DISCUSSION TITLE
Slicing Extension for UGS

### DISCUSSION THREAD

**Discussion by MuhammadZeeshan380** [winder/Universal-G-Code-Sender#2448] [Help]
I wanted to ask if there is any slicing extension for UGS. recently I used Mach3 with my printer and it had a slicing extension which got me wondering if there is one for UGS in any of the branches or something. PS I'm a complete amateur here so take my words with a grain of salt.

**Comment by breiler** [comment]
No, UGS has no features for 3D printers.
